src/unittest/test_keycode.cpp

```diff
@@ -111,7 +111,7 @@ void TestKeycode::testCompare()
 {
 	// Basic comparison
 	UASSERT(KeyPress("5") == KeyPress("KEY_KEY_5"));
-	UASSERT(!(KeyPress("5") == KeyPress("KEY_NUMPAD5")));
+	UASSERT(!(KeyPress("5") == KeyPress("KEY_NUMPAD_5")));
```

Comment:
No need to rollback this, it was broken

Reply:
So why does it not fail for me then?
```
; ./bin/minetest --run-unittests |[2] grep -i TestKeyCode
======== Testing module TestKeycode
======== Module TestKeycode passed (0 failures / 3 tests) - 12ms
```

Reply:
Because it's a bad test. `KEY_NUMPAD_5` does not exist.

Reply:
I think it should be `UASSERT(KeyPress("5") == KeyPress("KEY_NUMPAD5"));`, because we're changing the behavior.

Reply:
So, uh, should I have a failing test case with this commit or not? https://github.com/minetest/minetest/commit/e41f8365e14eced57f2390603b0e45ce3845eb84

Reply:
I get it now, numpad keys are handled different on purpose and this test was supposed to test for that.